### PR TITLE
Adding much more context about default serializers

### PR DESCRIPTION
**Pull request by pete-the-pete**

### CODE REVIEW

File: content/ember-data/v3/default-serializer.md [hunk 1]
```diff
@@ -5,31 +5,44 @@ until: '4.0.0'
...
 
 ## clearing these deprecations
+
 More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
 
 If a specific model type requires custom serialization, a type-specific serializer can be created. A single `application` serializer can be used a for any model types not requiring custom serialization. To define a type-specific serializer, create an `app/serializers/[type].js` with the following:
 
 ```js
-    import { JSONAPISerializer } from '@ember-data/serializers';
+    import RESTSerializer from '@ember-data/serializers/rest';
```
Comment (rwjblue) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
    import RESTSerializer from '@ember-data/serializer/rest';
```

File: content/ember-data/v3/default-serializer.md [hunk 2]
```diff
@@ -5,31 +5,44 @@ until: '4.0.0'
...
+    export default class UserSerializer extends RESTSerializer {
       /*custom code*/
     }
 ```
 
 Defining a serializer for the entire application can be done by adding the file `app/serializers/application.js` with the following:
 
 ```js
-    export { JSONAPISerializer } from '@ember-data/serializers';
+    export { default } from '@ember-data/serializers/json-api';
```
Comment (rwjblue) [inline content/ember-data/v3/default-serializer.md]:
```suggestion
    export { default } from '@ember-data/serializer/json-api';
```

Comment (runspired) [reply content/ember-data/v3/default-serializer.md]: confirm this should be done

File: content/ember-data/v3/default-serializer.md (hunk 1 above)
Comment (runspired) [reply content/ember-data/v3/default-serializer.md]: confirm this should be done

**Comment by pete-the-pete** [comment]
> I believe that the package we publish is `@ember-data/serializer` (not `@ember-data/serializers`), @runspired can you confirm?

I think that's correct? https://github.com/emberjs/data/tree/master/packages/serializer

**Pull request merged**